config/matrix-api-v1.yml

```diff
@@ -0,0 +1,299 @@
+              schema:
+                $ref: '#/components/schemas/MatrixErrorResponse'
+  /filters:
+    get:
+      summary: Get a list of permitted filters
+      description: >
+        Get a list of filters that a POST to /matrix will accept.
+      responses:
+        '200':
+          description: 'Successfully return filter list.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MatrixFilters'
+  /filters/{filter_name}:
+    get:
+      summary: 'Get a description of a filter'
+      description: >
+        Get detailed information about a filter that can be applied
+        to the HCA expression data.
+      parameters:
+        - name: filter_name
+          schema:
+            type: string
+          required: True
+          in: path
+          description: 'Filter to describe.'
+      responses:
+        '200':
+          description: "Filter found."
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/NumericFieldDetail'
+                  - $ref: '#/components/schemas/CategoricalFieldDetail'
+        '404':
+          description: 'Filter not found.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MatrixErrorResponse'
+  /fields:
+    get:
+      summary: Get a lsit of available metadata fields
+      description: >
+        Get a list of metadata field names that a POST to 
+        /matrix will accept.
+      responses:
+        '200':
+          description: 'Successfully return field list.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MatrixFields'
+  /fields/{field_name}:
+    get:
+      summary: 'Get a description of a metadata field'
+      description: >
+        Get detailed information about a an available metadata field.
+      parameters:
+        - name: field_name
+          schema:
+            type: string
+          required: True
+          in: path
+          description: 'Field to describe.'
+      responses:
+        '200':
+          description: "Field found."
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/NumericFieldDetail'
+                  - $ref: '#/components/schemas/CategoricalFieldDetail'
+        '404':
+          description: 'Field not found.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MatrixErrorResponse'
+components:
+  schemas:
+    FilterOperation:
+      description: 'Comparison operations allowed in matrix filters.'
+      type: string
+      enum:
+        - '='
+        - '!='
+        - '>'
+        - '<'
+        - 'in'
+    MatrixFormat:
+      description: 'Format for the output matrix.'
+      type: string
```

Comment:
This was previously an enumeration.

Reply:
Do you think it should be an enumeration defined in the API or rather described the response to a `/formats` endpoint?